rust/arrow/README.md

```diff
@@ -21,10 +21,62 @@
 
 [![Coverage Status](https://coveralls.io/repos/github/apache/arrow/badge.svg)](https://coveralls.io/github/apache/arrow)
 
+This crate contains a native Rust implementation of the [Arrow columnar format](https://arrow.apache.org/docs/format/Columnar.html). It uses nightly Rust.
+
+## Developer's guide
+
+Here you can find general information about this crate's content and its organization.
+
+### DataType, Field, Schema and RecordBatch
+
+Every array in Arrow has a data type and a null bitmap, that specifies whether each value is null or not.
```

Comment:
The null bitmap is optional (for arrays that do not contain any null values) but this makes it sound like that always has to be one.